componets/buttons/buttons.css

```diff
@@ -0,0 +1,106 @@
+.btn {
+  margin: 8px;
+  padding: 15px;
+  border-radius: 4px;
+  font-size: 1rem;
+  font-weight: 600;
+  border: none;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.btn-primary {
+  background-color: var(--primary-color);
+  color: var(--background-color);
+}
+.btn-primary:hover {
+  background-color: var(--secondary-color);
+}
+.btn-success {
+  background-color: var(--success-green-color);
+  color: var(--background-color);
+}
+.btn-success:hover {
+  background-color: #34d399;
```

Comment:
can be done using variables.

Reply:
yes did this change thanks for the suggestion